motr/io_nw_xfer.c

```diff
@@ -2181,18 +2191,33 @@ static int nw_xfer_req_dispatch(struct nw_xfer_request *xfer)
 			m0_op_io_to_rpc_map(ioo,
 					&irfop->irf_iofop.if_fop.f_item);
 
-			if (rc != 0)
-				goto out;
-			m0_atomic64_inc(&instance->m0c_pending_io_nr);
-			if (ri_error == 0)
-				M0_CNT_INC(nr_dispatched);
-			else if (post_error == 0)
-				post_error = ri_error;
+			if (rc != 0) {
+				/* This will only occur for rpc_bulk error,
+				 * continue dispatch operations after updating
+				 * the error response */
+				ti->ti_rc = ti->ti_rc ?: rc;
+				xfer->nxr_rc = xfer->nxr_rc ?: rc;
+				bulk_error = rc;
+				rc = 0;
+			}
```

Comment:
Move else to the same line as that of above closing brace.

Reply:
done